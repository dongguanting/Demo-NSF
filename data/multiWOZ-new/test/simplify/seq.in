i need a train leaving monday and arriving by 19:00
i have to leave from peterborough
just one
stansted airport after 11:45
departing stevenage after 19:45
when does the tuesday train leave
its from cabridge to london liverpool street
after 20:15
i 'm looking for a train to cambridge
from broxbourne on tuesday
id like to get there by 1645
yes one ticket
i need a train to cambridge early
departing from ely
to leicester from cambridge on saturday
i mean 20:45
i need to reach by 19:30
what trains depart from london kings cross
saturday to cambridge
the train should arrive by 14:00
booking for 5 pelase
i am leaving after 1630 on monday
11:15 arrival
i need to arrive at 12:30 on wedneday
leaving birmingham for cambridge
i need to be in cambridge by 12:30
saturday
i need to leave cambridge after 17:30
to stansted airport saturday
i would like a train leaving cambridge monday
i need three train seats
is there a train after 5 for liverpool street london
this friday
which train goes to cambridge on thursday
that would be correct
cambridge is the station i am travelling from
birmingham new street
traveling on monday
just one raffle please
leave london liverpool street
its wednesday
i 'd like to find a training leaving cambridge on monday
im leaving at 5:15
need to leave cambridge on sun
i search for the 11:45 stevenage train
thu train departures for stansted airport
any trains leaving from norwich
i am going to cambridge on tuesday
norwich on monday
book it for 4 people
going from cambridge to stansted
at 13:145
one
i am travelling from cambridge to bishops stortford by train
one only
london king cross arrive before 16:45
train to cambridge please
i 'm looking for a train from london on friday after 20:00
train departing from cambridge
trains from cambridge on sunday
will it arrive in stevenage by 9:00
i will be going to liverpool street on sunday
i need a train to cambridge
does train depart broxbourne station
sunday
after 13:45
tell me about a train from cambridge to broxbourne
may i have the price and id of the first train after 19:45
i am going from cambridg to the train airport
need train to cambridge
ii will depart at 800 for cambridge
sounds fine can i purchase 6 tickets
oh i 'm looking for cambridge - norwich train
sunday
i have to get there by 12:30
one ticket
yes its sunday
what about 17:15 and later
i need 2 so please book it
from cambridge on monday 12:45
which train leaves on thursday
i 'm looking to go to birmingham new street leaving on thursday
from cambridge and i should arrive by 17:15
cambridge to london liverpool
arrive at 15:45 on tuesday
i want tickets departing saturday from cambridge
i want to go to kings lynn
i need to get to cambridge from stansted airport
i would like to travel today after 7:30 pm from london to liverpool
train to ely please
thursday
at 14:15
cambridge to broxbourne
saturday
by 08:30
yes be fine please book for 6 people
find a train to norwich for monday
i want to leave after 17:00 and goes to cambridge
13:00 is when i need to arrive
for 4 people
the departure is from cambridge
need morning train from cambridge
id like to leave kings lynn by monday and arrive in cambridge by 1445
i am looking for a train towards london kings cross that departs after 08:15
evening train from cambridge no earlier than 20:45
liverpool street in london
yes thursday
taking train on wednesday after 16:00
kings lynn
its 9:45
id like to leave on sunday
i think wednesday birmingham new street
may i know what would be the cost
the train will arrive by 21:30
i need a train that leaves cambridge on tuesday
the train that travels from cabridge to stansted airport on tuesday and arrives closest to 21:30
what train leaves early on wednesday
can i book for 8 people and get a reference number
where 's the train
from cambridge to kings cross
want to leave from broxbourne after 12:30
i am looking for a train to broxbourne after 15:15
find train on sunday
is there a train for tuesday
heading to cambridge from stevenage after 10:45
i would like a train for wednesday
i 'd like a train to liverpool street that leaves after 10
leaving friday from cambridge
which trains are departing from broxbourne
to cambridge
by 17:15 i need to arrive
wednesday
does it arrive by 17:15
train to cambridge
ely this friday
09:55 is when i need to leave
saturday at 8:45
i need a ticket from cambridge on thursday at 08:45
leave after 20:15
one
cambridge from norwich
arrive cambridge by 14:15
two
please tell me about trains to cambridge on friday
train from cambridge
how do i get to standsted airport
i 'd like thursday
departure time after 16:15
i am looking for tickets for two people for the train on tuesday arriving at 10:30 am with final destination broxbourne and leaving cambridge
i will reach on monday at 14:15
monday after 12:00
leicester
i 'm going to cambridge monday
one
ticket to leaving cambridge to peterborough by 21:45
i need train from liverpool station wednesday
need to arrive by 14:45
wednesday
yes 5 people
which train departs from cambridge to peterborough
kings lynn
i 'll leave from cambridge
train should leave after 10:15 to peterborough
cambridge to ely please
looking for train should arrive by 18:30 go to peterborough
leaving sunday from cambridge
looking for train going to cambridge
i want a train to cambridge
the user will leave from brookshite
take me from london liverpool street to cambridge
one ticket please
tell me when the train arrives in london liverpool street
i need to leave on saturday
i need a train for thursday from cambridge to ely
trains from cambridge to leicester
monday
i 'd like  a train to peterborough
reserve 6 tickets next train after 11:15
sunday after 9:00
cambridge
i 'll be leaving from london liverpool street
departure time after 9pm
i want a time that arrives in king lynn at 8:00
trains to cambridge
sunday arriving at 13:00
need a train to broxbourne
ill need a train to london kings cross on wednesday
i need to leave at 18:00
find train that departs from peterborough arrives in cambridge by 8:45
i would like to leave saturday from london liverpool street
any trains from cambridge to ely
book it for 3 people and get me a reference number please
from bishops stortford to cambridge
wednesday after 08:15
the user wants a train that leaves on sunday
wednesday
i need to arrive by 18:15
i`ll go from cambridge to broxbourne
i am looking for a train to take 5me to cambridge
by 9:15 on thursday from stevenage to cambridge
i need to leave leicester by 20:30
need train out king lynn
leaving on tuesday
train to cambridget after 930
i need a train going to liverpool street on saturday
train from cambridge after 15:30
cambridge
cambridge to peterborough
yes book it for 1 person and provide the reference number
looking for train leaving  after 1145
stansted airport to cambridge friday
the offer works for him and he asks for a 7 train people
birmingham new street after 14:15
looking for a train to cambridge around 10:15
i want to go by train from cambridge to norwich
any trains leaving cambridge thursday
broxbourne
the one arriving at 3pm
six
i need a train to peterborough from cambridge on sunday
leave after 18:15
norwich
leave after 14:30
need to book departing cambridge after 9:15
which train depart from london liverpool street  on thursday to cambridge
i have to catch a train to cambridge
ok when was the train comes from kings lynn
leaving thursday
after 10:00
looking for train
thursday from cambridge
trains to peterborough
i want to go here i 'll leave after 2 pm
travel on tuesday
a train from stevenage to cambridge
any departing tuesday and arriving by 9:00
i want to leave for cambridge on wednesday
london kings cross is where i 'm coming from 
train from london to kings cross
the user wants to arrive in cambridge at 17:00 on a tuesday
he needs a train that passes by that street after 20:30
need something closer to 20:30
i want to go sunday
can i get a ticket for 8
i want 2 tickets
does it leave friday
train from ely  to camebridge
friday arrive by 14:15
the user 's going from stevenage to cambridge
yes book it for 8 people and i will need the reference number
stevenage saturday
i 'll leave after 12:15
need train for friday arrive by 19:00
cambridge
stansted airport
to cambridge leaving london liver pool street
the user needs a train from cambridge to broxbourne
i will travel on monday
can i get a train from cambridge to bishops stortford
thursday after 20:15
need a train that leaves cambridge after 21:00 thursday
looking for train to cambridge saturday
of course i want this saturday does it have
at or before 13:45
need a train that leaves wednesday with an arrival of 12:30
cambridge to liverpool street
he wants a tuesday train for leicester
leave after 11:15
from leicester to cambridge after 11:15
find train from leicester saturday
to cambridge and leave after 12 pm
i am looking for a train from london kings cross and leaves on wednesday
i need to leave friday on train from london liverpool street
arrival time by 14:00
user: i need a train to cambridge
on tuesday and arrive by 13:30
he wants a sunday train that leaves after 21:45
i shall leave from cambridge to stansted airport
need to leave after 21:45
iâ€™m departing at 19:00
six tickets please
looking for a train leaving london kings cross arriving by 14:30
cambridge
book 7 people at 10 please
cambridge
on a thursday please
arrive by 17:45
i 'd like a train leaving cambridge on sunday
going to leicester
i will leave after 10:15
looking for train leaving bishops stortford to cambridge
saturday
at 8:15
need a train from london kings cross to cambridge
is there a sunday train schedule
looking for train leaving stevenage
cambridge on friday
i will leave after 13:30
