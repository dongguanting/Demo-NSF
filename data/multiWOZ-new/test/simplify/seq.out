O O O O O B-day O O O B-arrive
O O O O O B-depart
O B-people
B-depart I-depart O B-leave
O B-depart O B-leave
O O O B-day O O
O O O O B-depart I-depart I-depart
O B-leave
O O O O O O O B-dest
O B-depart O B-day
O O O O O O B-arrive
O B-people O
O O O O O B-dest O
O O B-depart
O B-dest O B-depart O B-day
O O B-leave
O O O O O B-arrive
O O O O B-depart I-depart I-depart
B-day O B-dest
O O O O O B-arrive
O O B-people O
O O O O B-leave O B-day
B-arrive O
O O O O O B-arrive O B-day
O B-depart O B-dest
O O O O O B-dest O B-arrive
B-day
O O O O B-depart O B-leave
O B-dest I-dest B-day
O O O O O O B-depart B-day
O O B-people O O
O O O O O B-leave O B-dest I-dest I-dest
O B-day
O O O O B-depart O B-day
O O O O
B-depart O O O O O O O
B-dest I-dest I-dest
O O B-day
O B-people O O
O B-depart I-depart I-depart
O B-day
O O O O O O O O B-depart O B-day
O O O B-leave
O O O B-depart O B-day
O O O O B-arrive B-dest O
B-day O O O B-dest I-dest
O O O O B-depart
O O O O B-dest O B-day
B-depart O B-day
O O O B-people O
O O B-depart O B-dest
O B-leave
B-people
O O O O B-depart O B-dest I-dest O O
B-people O
B-dest I-dest I-dest O O B-arrive
O O B-dest O
O O O O O O O B-depart O B-day O B-leave
O O O B-depart
O O B-depart O B-day
O O O O B-dest O O
O O O O O B-dest I-dest O O
O O O O O B-dest
O O O B-depart O
B-day
B-leave I-leave
O O O O O O B-depart O B-dest
O O O O O O O O O O O O B-leave
O O O O B-depart O O B-dest I-dest
O O O B-dest
O O O O B-leave O B-dest
O O O O O B-people O
O O O O O B-depart O B-dest O
B-day
O O O O O O B-arrive
B-people O
O O B-day
O O B-leave O O
O O B-people O O O O
O B-depart O B-day B-arrive
O O O O B-day
O O O O O O B-dest I-dest I-dest O O B-day
O B-depart O O O O O B-arrive
B-depart O O O
O O O O B-day
O O O O B-day O B-depart
O O O O O B-dest I-dest
O O O O O B-dest O B-depart I-depart
O O O O O O O O O O B-depart O B-dest
O O B-dest O
B-day
O B-arrive
B-dest O B-depart
B-day
O B-leave
O O O O O O B-people O
O O O O B-depart O B-day
O O O O O B-leave O O O B-dest
B-arrive O O O O O O
O B-people O
O O O O B-depart
O O O O B-depart
O O O O B-depart I-depart O B-day O O O B-dest O B-arrive
O O O O O O O B-dest I-dest I-dest O O O B-leave
O O O B-depart O O O B-leave
B-dest I-dest O O
O B-day
O O O B-day B-leave I-leave
B-dest I-dest
O B-arrive
O O O O O B-day
O O B-day B-dest I-dest I-dest
O O O O O O O O
O O O O O B-arrive
O O O O O O O O B-day
O O O O O O O B-dest I-dest O B-day O O O O B-arrive
O O O O O B-day
O O O O B-people O O O O O O
O O O B-depart
O B-depart O B-dest I-dest
O O O O B-depart O B-leave
O O O O O O O B-dest O B-leave
O O O B-day
O O O O O B-day
O O B-dest O B-depart O B-leave
O O O O O O B-day
O O O O O O B-dest I-dest O O O B-leave
O B-day O B-depart
O O O O O B-depart
O B-dest
O B-arrive O O O O
B-day
O O O O B-arrive
O O B-dest
B-depart O B-day
B-leave O O O O O O
B-day O B-leave
O O O O O O O B-day O O
O O B-leave
B-people
B-dest O B-depart
O O O B-arrive
B-people
O O O O O O B-dest O B-day
O O B-depart
O O O O O B-dest I-dest
O O O B-day
O O O B-leave
O O O O O O B-people O O O O O B-day O O B-arrive O O O O B-dest O O B-depart
O O O O B-day O B-arrive
B-day O B-leave
B-dest
O O O O B-dest B-day
B-people
O O O B-depart O B-dest O B-arrive
O O O O B-depart O B-day
O O O O B-arrive
B-day
O B-people O
O O O O B-depart O B-dest
B-dest I-dest
O O O O B-depart
O O O O B-leave O B-dest
B-depart O B-dest O
O O O O O O B-arrive O O B-dest
O B-day O B-depart
O O O O O B-dest
O O O O O B-dest
O O O O O B-depart
O O O B-depart I-depart I-depart O B-dest
B-people O O
O O O O O O O B-dest I-dest I-dest
O O O O O B-day
O O O O O B-day O B-depart O B-dest
O O B-depart O B-dest
B-day
O O O O O O B-dest
O B-people O O O O B-leave
B-day B-leave I-leave
B-dest
O O O O O B-depart I-depart I-depart
O O O B-leave
O O O O O O O B-dest I-dest O O
O O B-depart
B-day O O B-arrive
O O O O B-dest
O O O O O B-dest I-dest I-dest O B-day
O O O O O B-arrive
O O O O O B-depart O O B-dest O B-arrive
O O O O O B-day O O O O
O O O B-depart O B-dest
O O O B-people O O O O O O O O
O B-depart I-depart O B-dest
B-day O B-leave
O O O O O O O O B-day
B-day
O O O O O B-arrive
O O O B-depart O B-dest
O O O O O O O O O O B-dest
O O O B-day O B-depart O O
O O O O B-depart O B-leave
O O O B-depart I-depart
O O B-day
O O B-dest O O
O O O O O O B-dest I-dest O B-day
O O B-depart O B-leave
B-dest
B-depart O B-dest
O O O O B-people O O O O O O
O O O O O B-leave
B-depart I-depart O B-dest B-day
O O O O O O O O O O B-people O O
B-dest I-dest I-dest O B-leave
O O O O O B-dest O B-arrive
O O O O O O O B-depart O B-dest
O O O B-depart B-day
B-dest
O O O O B-arrive
B-people
O O O O O B-dest O B-depart O B-day
O O B-leave
B-depart
O O B-leave
O O O O B-depart O O
O O O O B-depart I-depart I-depart O B-day O B-dest
O O O O O O O B-dest
O O O O O O O B-depart I-depart
O B-day
O B-leave
O O B-depart
B-day O B-depart
O O B-dest
O O O O O O O O O B-leave I-leave
O O B-day
O O O B-depart O B-dest
O O B-day O O O B-arrive
O O O O O B-dest O B-day
B-depart I-depart I-depart O O O O O O
O O B-depart O I-depart O
O O O O O O B-dest O B-arrive O O B-day
O O O O O O O O I-depart O O
O O O O B-leave
O O O O B-day
O O O O O O B-people
O O B-people O
O O O B-day
O O B-depart O B-dest
B-day O O B-arrive
O O O O O B-depart O B-dest
O O O O B-people O O O O O O O O
B-depart B-day
O O O O B-leave
O O O B-day O O B-arrive
B-depart
B-dest I-dest
O B-dest O B-depart I-depart I-depart I-depart
O O O O O O B-depart O B-dest
O O O O B-day
O O O O O O B-depart O B-dest I-dest
B-day O B-leave
O O O O O B-depart O B-leave B-day
O O O O B-dest B-day
O O O O O B-day O O O
O O O B-arrive
O O O O O B-day O O O O B-arrive
B-depart O B-dest I-dest
O O O B-day O O B-depart
O O B-leave
O O O B-dest O O
O O O O B-day
O B-dest O O O B-leave I-leave
O O O O O O O B-depart I-depart I-depart O O O B-day
O O O O B-day O O O B-depart I-depart I-depart
O O O B-arrive
O O O O O O B-dest
O B-day O O O B-arrive
O O O O O O O O B-leave
O O O O B-depart O B-dest I-dest
O O O O B-leave
O O O B-leave
B-people O O
O O O O O B-depart I-depart I-depart O O B-arrive
B-dest
O B-people O O B-leave O
B-depart
O O B-day O
O O B-arrive
O O O O O O B-depart O B-day
O O B-dest
O O O O B-leave
O O O O B-depart I-depart O B-dest
B-day
O B-arrive
O O O O B-depart I-depart I-depart O B-dest
O O O B-day O O
O O O O B-depart
B-dest O B-day
O O O O B-leave
